src/sentry/incidents/logic.py

```diff
@@ -1059,13 +1059,22 @@ def create_alert_rule_trigger_action(
         if target_type != AlertRuleTriggerAction.TargetType.SPECIFIC:
             raise InvalidTriggerActionError("Must specify specific target type")
 
-        channel_id = get_alert_rule_trigger_action_integration_object_id(
-            type.value, trigger.alert_rule.organization, integration.id, target_identifier
-        )
-
         # Use the channel name for display
         target_display = target_identifier
-        target_identifier = channel_id
+
+        if type == AlertRuleTriggerAction.Type.PAGERDUTY:
```

Comment:
@ceorourke I think we could move this logic into `get_alert_rule_trigger_action_integration_object_id` and make a new function that's like `get_alert_rule_trigger_action_pagerduty_service_id` and put that in there. We should strive to be consistent with what we already have.